stable/postgresql/templates/configmap.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "postgresql.fullname" . }}
+  labels:
+    app: {{ template "postgresql.name" . }}
+    chart: {{ template "postgresql.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+data:
+  {{- if .Values.metrics.customMetrics }}
+  custom-metrics.yaml: {{ toYaml .Values.metrics.customMetrics | quote }}
+  {{- end }}
```

Comment:
I don't know how policies are, but I think it would be good to finish with a newline here. Does someone know if there is such a policy?

Reply:
There is a newline in the existing secret.yaml but its not enforced by helm lint